trace/trace.go

```diff
@@ -289,6 +295,14 @@ func (s *Span) SpanContext() SpanContext {
 	return s.spanContext
 }
 
+// Tracestate returns the Tracestate of the span context.
+func (s *Span) Tracestate() *Tracestate {
```

Comment:
Why isn't this a method on SpanContext?

Reply:
The Span.SpanContext implementation will construct a new SpanContext if it is nil, which makes it hard for consumer to tell the difference whether Tracestate is nil in the existing SpanContext because it is nil or because the SpanContext is nil.